apps/mobile/src/components/ChatDetailScreen/components/UserAvatarDetail.tsx

```diff
@@ -0,0 +1,66 @@
+import {BackHandler, Modal, useWindowDimensions} from 'react-native'
+import {userAvatarDetailAtom} from '../../../state/chat/atoms/showUserAvatarDetailAtom'
+import {useAtom} from 'jotai'
+import {Stack} from 'tamagui'
+import {useSafeAreaInsets} from 'react-native-safe-area-context'
+import IconButton from '../../IconButton'
+import closeSvg from '../../images/closeSvg'
+import {ImageZoom} from '@likashefqet/react-native-image-zoom'
+import {useFocusEffect} from '@react-navigation/native'
+import React from 'react'
+
+function UserAvatarDetail(): JSX.Element {
+  const {width} = useWindowDimensions()
+  const {bottom, top, right, left} = useSafeAreaInsets()
+  const [userAvatarDetail, setUserAvatarDetail] = useAtom(userAvatarDetailAtom)
+
+  useFocusEffect(
+    React.useCallback(() => {
+      const onBackButtonPressed = (): boolean => {
+        setUserAvatarDetail(undefined)
+        return true
+      }
+
+      return BackHandler.addEventListener(
+        'hardwareBackPress',
+        onBackButtonPressed
+      ).remove
+    }, [setUserAvatarDetail])
+  )
+
+  return (
+    <Modal transparent animationType="fade" visible={!!userAvatarDetail}>
+      <Stack
+        f={1}
+        pt={top}
+        pb={bottom}
+        pl={left}
+        pr={right}
+        jc={'center'}
+        ai={'center'}
+        bc={'rgba(0,0,0,0.8)'}
+      >
+        <Stack pos={'absolute'} top={top} right={'$4'} zIndex={'$100'}>
+          <IconButton
+            icon={closeSvg}
+            onPress={() => {
+              setUserAvatarDetail(undefined)
+            }}
+          />
+        </Stack>
+        {userAvatarDetail && (
+          <ImageZoom
```

Comment:
Zoom does not work on android emulator (holding _cmd_ to use pinch gesture). Am I doing something wrong?